src/App.vue

```diff
@@ -550,18 +551,26 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     // NB: will throw if API error
     const response = await this.getCurrentUser()
     // NB: just save email for now
-    let email
+    let email, phone
```

Comment:
This will cause a conflict as I have refactored it (to handle account info and org info).

Ah, c'est la vie.

Reply:
ah, sorry :( Hopefully minor.